db/models/orders.js

```diff
@@ -135,6 +168,12 @@ const updateOrder = async ({ id, ...fields }) => {
             `,
       Object.values(fields)
     );
+    if (!order) {
+      throw {
+        name: "NoExistingInformation",
+        message: "No order currently exist.",
```

Comment:
we could maybe build a better message here: "The order you tried to update does not exist"

Reply:
Done.